src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -157,10 +157,37 @@ export default {
   },
 
   async mounted() {
-    await this.openStream();
-    await this.initRecorder();
-    if (this.recorderState === RECORDER_STATE.INITIALIZING) {
-      await this.startRecording();
+    try {
+      await this.openStream();
```

Comment:
openStream called in different places, you should catch errors in `captureMedia` method

Reply:
done